infrastructure/src/main/java/mops/database/dto/AntwortDto.java

```diff
@@ -0,0 +1,37 @@
+package mops.database.dto;
+
+import java.util.HashSet;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+
+@AllArgsConstructor
+@Data
+@Table("antwort")
+public class AntwortDto {
+  @Id
+  Long id;
+  String textantwort;
+  @Column("antwort")
+  Set<AuswahlDto> ausgewaehlt;
+
+  public static AntwortDto createTextantwort(String text) {
+    return new AntwortDto(null, text, null);
+  }
+
+  public static AntwortDto createMultipleChoiceAntwort() {
+    return new AntwortDto(null, null, new HashSet<>());
+  }
+
+  public void addAntwortToMultipleChoice(AuswahlDto auswahl) {
+    ausgewaehlt.add(auswahl);
+  }
+
+  public void addAntwortenToMultipleChoice(Set<AuswahlDto> auswahlen) {
+    ausgewaehlt.addAll(auswahlen);
+  }
```

Comment:
I don't know if we need those methods. I think, saving of answers should be done in business logic (it's already done there). When you translate from business logic to DTOs, the new answers are saved automatically.

Reply:
for create methods: same as one file down
for other methods: are right now used to ensure that the right type of answer will be saved to database, but might be replaceable if modelservice will to this job